OpenSim/Simulation/Model/Model.cpp

```diff
@@ -661,9 +661,15 @@ void Model::createMultibodyTree()
 
     auto bodies = getComponentList<Body>();
     for (auto& body : bodies) {
-        _multibodyTree.addBody( body.getAbsolutePathName(),
-                                body.getMass(), false,
-                                const_cast<Body*>(&body) );
+        try {
+            _multibodyTree.addBody( body.getAbsolutePathName(),
+                                    body.getMass(), false,
+                                    const_cast<Body*>(&body) );
+        } catch (const std::runtime_error& e) {
+            cout << "Duplicate Body names are not permitted. Please update "
+                 << "your model's .osim file accordingly." << std::endl;
```

Comment:
Will do. Actually, I don't think this error-checking strategy is correct: OpenSim shouldn't be relying on Simbody to report that the body-absolute-path-name-uniqueness requirement has been violated; this check should be done in `Component::finalizeFromProperties()` or similar. Created #1876 to track.